docs/commands/rhoas_completion.md

```diff
@@ -0,0 +1,47 @@
+## rhoas completion
```

Comment:
Actually I will remove markdown files and generate them in separate PR

Reply:
This will obviously fail the build so I will generate docs as last step before merge :D